packages/iris-grid/src/asserts.tsx

```diff
@@ -0,0 +1,15 @@
+export function assertNotNull<T>(value: T | null): asserts value is T {
+  if (value == null) throw new Error('Value is null');
+}
+
+export function assertNotUndefined<T>(
+  value: T | undefined
+): asserts value is T {
+  if (value === undefined) throw new Error('Value is undefined');
+}
+export function assertNotNullNorUndefined<T>(
+  value: T | null | undefined
+): asserts value is T {
+  assertNotNull(value);
+  assertNotUndefined(value);
+}
```

Comment:
I don't think you need this. `assertNotNull` is already verifying the value is not undefined (since it's doing `value == null` instead of `value === null`).

Reply:
Yes, assertNotNull is verifying that value is not undefined, but TypeScript can't recognize that, so I need an explicit assertNotUndefined